### PR TITLE
E168. Adding test cases for get_number_of_taggable_answers - tag_prompt_deployment

### PR DESCRIPTION
**Pull request by ragesah**
Please set the title of your pull request in the following format: `[Project id/Independent study/Issue fix/...]. project name`, eg., `E904. Integrating courses and assignments on Expertiza with Moodle`

We suggest you including the below information in your pull request:
- A description of the changes proposed in the pull request.
- @mentions of the person or team responsible for reviewing proposed changes.
---
About Expertiza Bot
- If you have any questions about comments given by the expertiza-bot [(example)](https://github.com/expertiza/expertiza/pull/1877#issuecomment-762412918), you could create a comment in your pull request with the format `/dispute [UUID1] [UUID2]` [(example)](https://github.com/expertiza/expertiza/pull/1877#issuecomment-762430057) then the professor and TAs will be notified.
- [Here is a video](https://tinyurl.com/internet-bots) about how to communicate to expertiza-bot.

### CODE REVIEW

File: spec/models/tag_prompt_deployment_spec.rb [hunk 1]
```diff
@@ -1,10 +1,69 @@
 describe TagPromptDeployment do
 	let(:tag_dep) { TagPromptDeployment.new id: 1, tag_prompt: tp, tag_prompt_id: 1, question_type: "Criterion", answer_length_threshold: 5 }
+	let(:tag_dep1) { TagPromptDeployment.new id: 1, tag_prompt: tp, tag_prompt_id: 1, question_type: "Criterion", answer_length_threshold: nil, assignment_id: 1, assignment: assignment , questionnaire: questionaire}
```
Comment (akk5597) [inline spec/models/tag_prompt_deployment_spec.rb:3]: `id` for this should not be 1 right? As it's already being used by `tag_dep` in the previous line.

File: spec/models/tag_prompt_deployment_spec.rb [hunk 2]
```diff
@@ -1,10 +1,69 @@
...
 			expect(tag_dep.tag_prompt).to be(tp)
 		end
 	end
-end
\ No newline at end of file
+#get_number_of_taggable_answers calculates total taggable answers assigned to an user who participated in "tag review assignment".
+	describe '#get_number_of_taggable_answers' do
+		before(:each) do
+			questions_ids = double(1)
+			responses_ids = double(241)
```
Comment (akk5597) [inline spec/models/tag_prompt_deployment_spec.rb:24]: Line 23 and 24 are repeated at two places lines (38, 39) and lines (47, 48). Are all 3 needed?

Comment (ragesah) [reply spec/models/tag_prompt_deployment_spec.rb:24]: Yes, they have required else it gives error.

File: spec/models/tag_prompt_deployment_spec.rb (hunk 1 above)
Comment (ragesah) [reply spec/models/tag_prompt_deployment_spec.rb:3]: I don't think this will cause any issue as they are being used in a different method.

**Review by akk5597 — APPROVED** [review]

**Pull request merged**